### ISSUE TITLE
The corpus has 207 videos and not one of them is about a concept

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#411]
Every `kind: video` explainer on the site is `scope: domain`:

```
video · domain-scoped    207
video · concept-scoped     0
read  · domain-scoped    335
read  · concept-scoped   147
```

So a reader on `/c/backpropagation` is offered 3Blue1Brown's *Backpropagation, intuitively* labelled **"on Training"** — correct, and ADR-0018 makes the page say so before they click, but it is a resource about the neighbourhood rather than about the thing they are reading. That is the domain pass working exactly as designed (#386): it searched 117 domains, so it found 117 domains' worth of videos.

The first `--concepts` run produced 135 candidates across 41 targets before its quota went to the bug fixed in #400. **22 of them are worth shipping**, verified through oEmbed:

```
22 verified, 0 rejected → video-explainers.json
```

## The picks

| concept | author — title |
|---|---|
| `backpropagation` | 3Blue1Brown — *Backpropagation calculus, Chapter 4* |
| `attention` | StatQuest — *Attention for Neural Networks, Clearly Explained* |
| `beam-search` | DeepLearningAI — *C5W3L03 Beam Search* |
| `bias-variance-tradeoff` | StatQuest — *Machine Learning Fundamentals: Bias and Variance* |
| `bayesian-inference` | 3Blue1Brown — *Bayes theorem, the geometry of changing beliefs* |
| `actor-critic` | Google DeepMind — *DeepMind × UCL RL: Policy-Gradient and Actor-Critic* |
| `advantage-estimation` | Pieter Abbeel — *L3 Policy Gradients and Advantage Estimation* |
| `batch-normalization` | DeepLearningAI — *Why Does Batch Norm Work?* |
| `activation-checkpointing` | PyTorch — *FSDP activation checkpointing* |
| `activation-steering` | Hugging Face — *Steering LLM Behavior Without Fine-Tuning* |
| `autoregressive-decoding` | Hugging Face — *Transformer models: Decoders* |
| `action-chunking` | Hugging Face — *ALOHA and ACT* |
| `byte-pair-encoding` | Greg Durrett — *Word Piece And Byte Pair Encoding* |
| `aleatoric-uncertainty` | Maziar Raissi — *Aleatoric vs Epistemic Uncertainty* |
| `adversarial-example` | AI Coffee Break — *Adversarial Machine Learning explained* |
| `approximate-nearest-neighbor` | ritvikmath — *Approximate Nearest Neighbors* |
| `attention-sink` | bycloud — *Attention Sink* |
| `adam` | Sourish Kundu — *Who's Adam and What's He Optimizing?* |
| `autoencoder` | AIEngineering — *Autoencoder Explained* |
| `active-learning` | Computerphile — *Active (Machine) Learning* |
| `batch-size` | deeplizard — *Batch Size in a Neural Network explained* |
| `activation-function` | deeplizard — *Activation Functions in a Neural Network explained* |

**Two of these argue against the ranking, which is worth recording.** `active-learning`'s pick scored **0.55 and ranked second**, behind a classroom-pedagogy video — the right answer was already in the list and the scorer could not see it, which is #401's premise confirmed from data that already existed. `batch-size`'s pick scored **0.35**, exactly the threshold, and is plainly the best of its three. The score sorts a list for a person; it does not decide.

## What was left out, and why

16 of the 38 targets produced nothing worth attaching, and ADR-0018 is explicit that attaching one anyway is the invented-reference rule wearing a different field name:

- **Homonyms with nothing usable** — `ablation` (all four candidates cardiac), `alibi` (a legal-English dictionary entry), `audio-visual-learning` (AV systems), `audio-generation` (RAG, Pokémon cries), `backdoor-attack` (the XZ software backdoor).
- **Already served** — `agent-memory` and `catastrophic-forgetting` each already carry a video that is squarely about them; a second is noise.
- **Essentially unwatched** — `beam-search-tradeoffs` (16, 479, 20, 16 views), `benchmark-contamination` (5, 81), `adamw` (727), `adaptive-computation`.
- **Nothing that teaches the concept** — `adversarial-training`, `agent-evaluation`, `arithmetic-intensity`, `any-to-any-model`, `agentic-coding`.

## Acceptance criteria

- [ ] 22 concept-scoped `kind: video` explainers added, titles and authors exactly as oEmbed reported them so `check:explainers` agrees by construction
- [ ] Concept-scoped entries render before domain-scoped ones, so the most specific resource is offered first
- [ ] No node gains a second video about the same thing it already had
- [ ] `validate:content` and `check:explainers` both pass in CI, where `youtube.com` is reachable

Refs #61, #384, #386, #400, #401, ADR-0017, ADR-0018.
